app/src/main/java/com/babylon/wallet/android/presentation/ui/RadixWalletPreviewTheme.kt

```diff
@@ -0,0 +1,19 @@
+package com.babylon.wallet.android.presentation.ui
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import com.babylon.wallet.android.designsystem.theme.RadixWalletTheme
+import com.babylon.wallet.android.presentation.ui.composables.actionableaddress.ProvideMockActionableAddressViewEntryPoint
+
+@Composable
+fun RadixWalletPreviewTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit,
+) {
+    ProvideMockActionableAddressViewEntryPoint {
```

Comment:
We could generalise this later, meaning that we could provide a place for mock classes during preview rendering. (And not only mock actionable address related)

Reply:
💯 % agree!